### PR TITLE
[13.0][MIG] hr_holidays_validity_date

### PR DESCRIPTION
**Pull request by jarroyomorales**
Migration to v13 of `hr_holidays_validity_date`.

@AaronHForgeFlow @Nikul-Chaudhary @Saran440 @etobella @MiquelRForgeFlow

### CODE REVIEW

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by AaronHForgeFlow — APPROVED** [review]
Functional + code :+1:

**Comment by Saran440** [comment]
Functional Test.

Error when click day on dashboard calendar
Peek 2020-09-30 10-40

and Create Time Off
Peek 2020-09-30 10-43

**Comment by jarroyomorales** [comment]
@Saran440 Fixed! 

Also now i'm using odoo's `_check_leave_type_validity` core function in order to compute warnings.

**Comment by MiquelRForgeFlow** [comment]
black

**Review by Saran440 — APPROVED** [review]
Functional Test :+1:

File: hr_holidays_validity_date/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "HR holidays validity date",
+    "summary": """
+        Allow to define start and end date on holidays type.""",
+    "author": "ACSONE SA/NV," "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/hr",
```
Comment (Saran440) [inline hr_holidays_validity_date/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/hr-holidays",
```

**Review by Saran440 — COMMENTED** [review]
Code reviewed

File: hr_holidays_validity_date/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "HR holidays validity date",
+    "summary": """
+        Allow to define start and end date on holidays type.""",
+    "author": "ACSONE SA/NV," "Odoo Community Association (OCA)",
```
Comment (Saran440) [inline hr_holidays_validity_date/__manifest__.py]:
```suggestion
    "author": "ACSONE SA/NV, Odoo Community Association (OCA)",
```

**Comment by jarroyomorales** [comment]
@Saran440 Thank you!

**Pull request merged**